### DISCUSSION TITLE
Unable to establish a direct connection to the database from pgAdmin.

### DISCUSSION THREAD

**Discussion by bala-goflink** [supabase/supabase#26018] [Questions] [answered]
I've referred to the [documentation](https://supabase.com/docs/guides/database/connecting-to-postgres#connecting-with-pgadmin), and I'm attempting to establish a direct connection to my database from pgAdmin. This is so I can perform a bulk upload of 10 million records into my database. However, I'm encountering difficulties connecting to my database.

Additionally, I want to mention that I successfully executed a database migration using the Supabase CLI without any connection issues.

<img width="707" alt="Screenshot 2024-05-10 at 17 57 56" src="https://github.com/supabase/supabase/assets/98825322/3242a818-53c6-49d0-9c53-3af99d63cec2">

Supabase CLI version: 1.165.0
PgAdmin 4 version: 7.7
macOS version: 14.4.1

**Comment by GaryAustin1** [comment]
Check the SSL stuff at the end here: https://supabase.com/docs/guides/database/connecting-to-postgres

**Reply by bala-goflink** [reply]
Enabling SSL and providing the SSL root certificate doesn't make any difference.

**Reply by GaryAustin1** [reply]
Under windows... I loaded the latest pgAdmin... 
![image](https://github.com/supabase/supabase/assets/54564956/eb7a9ffd-0d74-4a17-9dc5-1ee12ba97921)
And connected immediately and can see everything.  
I did not do anything with SSL though.

**Reply by bala-goflink** [reply, marked as answer]
I installed the newest version of pgAdmin, Version 8.6, and that fixed the problem. Now I can successfully connect.
